index.html

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+
+    <!-- header and headline -->
+    
+    <section class="first-page">
+        <header>
+            <nav id="head">
+                <a href="#">Safar Ali</a>
+                <a href="#"><img src="./images/Union.png" alt="burger-icon"></a>
+            </nav>
+        </header>
+        <section class="headline">
+            <div class="name">
+                <h1>I'm Safar Ali <br> A thriving <br> Software Developer </h1>
+            </div>
+            <div class="para">
+                <p>I am a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+            <div class="text">
+                <p>Let us Connect</p>
+            </div>
+            <div class="icons">
+                <ul>
+                    <li><a href="#"><img src="./images/github.png" alt="github-page"></a></li>
+                    <li><a href="#"><img src="./images/Linkedin.png" alt="linkedin-page"></a></li>
+                    <li><a href="#"><img src="./images/twitter.png" alt="twitter-page"></a></li>
+                    <li><a href="#"><img src="./images/Vector.png" alt="vector-page"></a></li>
+                </ul>
+            </div>
+        </section>
+    </section>
+
+    <!-- Main-section -->
+
+    <main class="main">
+
+        <!-- page-2 -->
+
+        <section class="page-2">
+            <div class="picture">
+                <a href="#"><img src="./images/picture-1.png" alt="picture-1"></a>
+            </div>
+            <div class="heading">
+                <h1>Tonic</h1>
```

Comment:
Please use h2 here. Because h1 should be used only once. And you've used that in first page